sycl/doc/design/spirv-extensions/SPV_INTEL_joint_matrix.asciidoc

```diff
@@ -136,19 +181,70 @@ is spread across multiple invocations.
 
 Add _Joint Matrix_ to the definition of _Composite_.
 
-=== Matrix layout
+Add _Joint Matrix_ to the definition of _Concrete Type_.
+
+=== Joint Matrix Layout
 
-Add section 3.XX, Matrix layout.
+Add section 3.XX, Joint Matrix Layout.
+'Layout' indicates how the values of joint matrix are arranged in memory.
 
 [options="header"]
 |====
 2+^| Layout ^| Enabling capability 
-| 0 | *ColumnMajor*            |  *{capability_name}*
-| 1 | *RowMajor*               |  *{capability_name}*
-| 2 | *PackedA* +
-Suitable for VNNI instructions | *{capability_name}*
-| 3 | *PackedB* +
-Suitable for VNNI instructions | *{capability_name}*
+| 0 | *RowMajor*               |  *{main_capability_name}*
+| 1 | *ColumnMajor*            |  *{main_capability_name}*
+| 2 | *Packed* +
+Suitable for Vector Neural Network Instructions (VNNI) used in Intel AMX
```

Comment:
Seems odd to say "VNNI instructions are used by AMX and XMX". Maybe "VNNI instruction format"?

Reply:
Applied, thanks!